operator/gitops/argocd/pipeline-service/metrics-exporter/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ resources:
 images:
   - name: quay.io/redhat-appstudio/pipeline-service-exporter
     newName: quay.io/redhat-appstudio/pipeline-service-exporter
-    newTag: 7ffa67747cae05345dd1e9a274ec26c50d508068
+    newTag: fb7dd3a3bca5cde981c552e6ec50c8054c590506
```

Comment:
tag exists in https://quay.io/repository/redhat-appstudio/pipeline-service-exporter?tab=tags